demos/multi_channel_common/cpp/graph.hpp

```diff
@@ -18,6 +18,7 @@
 #include <inference_engine.hpp>
 
 #include <utils/common.hpp>
+#include <utils/performance_metrics.hpp>
```

Comment:
Why is it included?

Reply:
Accidentally, removed in commit https://github.com/openvinotoolkit/open_model_zoo/pull/2647/commits/b6eee922e828dbaa7a02b453dcf116fa96bb1ebc